ccmlib/dse_node.py

```diff
@@ -95,6 +95,8 @@ def start(self,
         if wait_other_notice:
             marks = [(node, node.mark_log()) for node in list(self.cluster.nodes.values()) if node.is_running()]
 
+        self.mark = self.mark_log()
```

Comment:
It's self.mark in node.py currently. We can hash out the fix on the main branch there, but symmetry is preferred for now.